storage_service/locations/models/s3.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import absolute_import
+# stdlib, alphabetical
+import logging
+import os
+
+# Core Django, alphabetical
+from django.db import models
+from django.utils.translation import ugettext as _, ugettext_lazy as _l
+
+# Third party dependencies, alphabetical
+import boto3
+import re
+
+# This project, alphabetical
+
+# This module, alphabetical
+from . import StorageException
+from .location import Location
+
+LOGGER = logging.getLogger(__name__)
+
+
+class S3(models.Model):
+    space = models.OneToOneField('Space', to_field='uuid')
+    endpoint_url = models.CharField(max_length=2048,
+        verbose_name=_l('endpoint_url'),
+        help_text=_l('Endpoint URL'))
+    access_key_id = models.CharField(max_length=64,
+        verbose_name=_l('access_key_id'),
+        help_text=_l('Access Key ID to authenticate'))
+    secret_access_key = models.CharField(max_length=256,
+        verbose_name=_l('secret_access_key'),
+        help_text=_l('Secret Access Key to authenticate with'))
+    region = models.CharField(max_length=64,
+        verbose_name=_l('Region'),
+        help_text=_l('Region'))
```

Comment:
Just noticed that the field labels in the HTML have underscores in them because of the verbose names given here. I think the verbose names should be changed: remove the underscores and in general make them "user-facing friendly". Relatedly, if the `help_text` value is the same as the verbose name, it should not be included. For region and endpoint_url, I think the help_text values should be changed into a descriptive/explanatory sentence